x-pack/metricbeat/module/aws/terraform.tf

```diff
@@ -1,5 +1,14 @@
 provider "aws" {
-  version = "~> 2.58"
+  version = "~> 3.33"
```

Comment:
Forced to bump the version to be able to use the `default_tags`:
- https://github.com/hashicorp/terraform-provider-aws/blob/v3.33.0/CHANGELOG.md